gcc/rust/rust-session-manager.cc

```diff
@@ -479,6 +543,32 @@ Session::enable_dump (std::string arg)
 void
 Session::parse_files (int num_files, const char **files)
 {
+  if (options.crate_name.empty ())
+    {
+      /* HACK: We use the first file to infer the crate name, which might be
+       * incorrect: since rustc only allows one file to be supplied in the
+       * command-line */
+      auto filename = "-";
+      if (num_files > 0)
+	filename = files[0];
+
+      auto crate_name = infer_crate_name (filename);
+      Error error ((Location ()), std::string ());
```

Comment:
Same nitpick here, feel free to ignore
```suggestion
      auto error = Error (Location (), std::string ());
```